solidity/computation-datasource/bitcoin/BitcoinBalanceExample.sol

```diff
@@ -17,7 +17,7 @@ contract BitcoinBalanceExample is usingOraclize {
         balance = parseInt(result, 8);
         emit LogBitcoinAddressBalance(balance);
     }
-    
+
     function getBalance(string memory _bitcoinAddress) public payable {
         oraclize_query("computation",["QmaMFiHXSqCFKkGPbWZh5zKmM827GWNpk9Y1EYhoLfwdHq", _bitcoinAddress]);
```

Comment:
computation archive is down

Reply:
Not according to Infura it isn't. That said, the bitcoin example is still failing for me. Will look into.

Reply:
Weird, it is stuck for me when I use `ipfs get ...`.

For me it is not really failing, I just don't receive any callback.